boards/arm/lpcxpresso55s16/Kconfig.defconfig

```diff
@@ -28,4 +28,16 @@ config GPIO_MCUX_LPC_PORT1
 
 endif # GPIO_MCUX_LPC
 
+config I2C_4
```

Comment:
Are these needed the (I2C_4 and SPI_8)?

Reply:
For now they are (due to https://github.com/zephyrproject-rtos/zephyr/blob/28e2bfc68e728410ee82f81cb6db42b3e7438dc7/soc/arm/nxp_lpc/lpc55xxx/soc.c#L64 and https://github.com/zephyrproject-rtos/zephyr/blob/28e2bfc68e728410ee82f81cb6db42b3e7438dc7/soc/arm/nxp_lpc/lpc55xxx/soc.c#L72).

Reply:
Gotcha, will send PR to change that